src/js/_enqueues/lib/comment-reply.js

```diff
@@ -303,11 +310,17 @@ window.addComment = ( function( window ) {
 		var postIdField     = getElementById( config.postIdFieldId );
 		var element, cssHidden, style;
 
+		var replyHeadingText = document.getElementsByClassName( config.commentReplyTitleInnerClass )[0];
+
 		if ( ! addBelowElement || ! respondElement || ! parentIdField ) {
 			// Missing key elements, fail.
 			return;
 		}
 
+		if ( 'undefined' === typeof replyTo) {
+			replyTo = replyHeadingText.textContent;
```

Comment:
As above. May be good to see if `replyHeadingText` exists just to be on the safe side.